src/Integration.Vsix/Integration.Vsix.csproj

```diff
@@ -338,51 +320,9 @@
   </Target>
   <!-- ********************************************************** -->
 
-  <!-- ********************************************************** -->
-  <!-- Generate the protobuf classes -->
-  <!-- ********************************************************** -->
-  <Target Name="GenerateProtobufClasses" BeforeTargets="PrepareForBuild" Outputs="$(ProjectDir)Protobuf\SonarlintDaemon.cs" Inputs="$(ProjectDir)Protobuf\sonarlint-daemon.proto">
-    <Message Importance="high" Text="Generating protobuf classes..." />
-    <Exec WorkingDirectory="$(ProjectDir)Protobuf" Command="build.bat &quot;$(NuGetPackageRoot)Grpc.Tools\1.4.1\tools\windows_x64&quot;" />
-
-    <!-- Make sure that any files we have just generated are included in the compilation -->
-    <ItemGroup>
-      <Compile Remove="Protobuf\*.cs" />
-      <Compile Include="Protobuf\*.cs" />
-    </ItemGroup>
-
-  </Target>
-
   <Import Project="Sdk.targets" Sdk="Microsoft.NET.Sdk" />
   <Import Project="$(VSToolsPath)\VSSDK\Microsoft.VsSDK.targets" Condition="'$(VSToolsPath)' != ''" />
 
-
-  <!-- ********************************************************** -->
-  <!-- Reference Grpc and System.Interactive.Async.dll -->
```

Comment:
We no longer need any of this nastiness to include the correct Grpc binaries and dependencies.

The resulting VSIX still includes the protobuf dll, but that is because it is included as a dependency of SonarQube.Client assembly. We don't need to do anything special to embed the correct protobuf assembly in the VSIX.